compiler/src/dmd/errors.d

```diff
@@ -722,7 +722,8 @@ private void verrorPrint(const(char)* format, va_list ap, ref ErrorInfo info)
 
     __gshared SourceLoc old_loc;
     auto loc = info.loc;
-    if (global.params.v.printErrorContext &&
+    if (global.params.v.errorPrintMode == ErrorPrintMode.printErrorContext &&
+        global.params.v.errorPrintMode != ErrorPrintMode.simpleError &&
```

Comment:
`global.params.v.errorPrintMode != ErrorPrintMode.simpleError` is redundant with `global.params.v.errorPrintMode == ErrorPrintMode.printErrorContext`

Reply:
Yes, makes sense. I think it's better to keep the `simpleError` case.

Reply:
done!